src/inference/graph/nodes.py

```
@@ -1,706 +1,240 @@
-    Parameters
-    ----------
-    state : GovOnGraphState
-        현재 graph state. `messages`와 `accumulated_context`를 읽는다.
-    planner_adapter : PlannerAdapter
-        graph config에서 closure로 주입되는 planner 어댑터.
-
-    Returns
-    -------
-    dict
-        `task_type`, `goal`, `reason`, `planned_tools`를 갱신한다.
+def make_session_load_node(session_store: "SessionStore"):
```

Ruff (0.15.9)

[warning] Unused function argument: `session_store`

(ARG001)